gulpfile.ts

```diff
@@ -50,21 +50,14 @@ gulp.task('test', ['build_server']);
 gulp.task('run_server', () => {
   nodemon({
     script: 'out/server.js',
-    ignore: 'out/'
+    ignore: ['out/']
```

Comment:
I think [] block we do not need now.
How do you think add [] block after we need multiple arguments?

Reply:
Thank you for review.
In Javascript, we could do it because there is no explicit types.
But after this patch, applying strong-type checking, the ```ignore``` parameter should be string[] type as per definition.
You can see the definition here: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/gulp-nodemon/index.d.ts#L39

Reply:
@yjaeseok I have to wait for your reply but I don't have enough time now. So, I'll merge this patch and then review this again (TBR)

Reply:
Oh sooorrry !